.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Please attach a sample .msapp to this Issue which demonstrates the issue you are experiencing. *(Please do not attach any file which may contain sensitive personal or company data.)*
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
You need an additional newline between the headers and descriptions or the description will appear side-by-side instead of below:
![image](https://user-images.githubusercontent.com/71468580/158447906-11530bb8-aa16-4d95-917d-c831b48775f6.png)

This applies to all the headers/descriptions.

Reply:
Good point, thanks for spotting it.